quarkus/service/src/test/java/org/apache/polaris/service/quarkus/admin/PolarisAuthzTestBase.java

```diff
@@ -87,9 +87,9 @@ public static class Profile implements QuarkusTestProfile {
     @Override
     public Map<String, String> getConfigOverrides() {
       return Map.of(
```

Comment:
IIUC, these need to map to entries in the Quarkus config file rather than the actual Polaris features?

Is there a way we can make that mapping programmatic rather than manual?

Reply:
But can we have it be programmatic? Is there a way to look up key:value pairs from the Quarkus config file?

Reply:
No, they do not form a set like an enum or something like that. The amount of available configuration options depends on the extensions being used.